server/controllers/articles.js

```diff
@@ -1 +1,129 @@
 import Article from "../models/Articles";
+export const getAllArticles = async (req, res, next) => {
+    try {
+        const articles = await Article.find();
+
+        return res.status(200).json({
+            success: true,
+            status: 200,
+            count: articles.length,
+            data: articles
+        });
+    } catch (err) {
+        return res.status(500).json({
+            success: false,
+            status: 500,
+            error: 'Server error'
+        });
+    }
+}
+
+export const createArticle = async (req, res, next) => {
+        const newArticle = new Article (
+            {
+                title: req.body.title,
```

Comment:
Could you please validate these inputs. You can use Joi or Validatorjs to validate them.